services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/unit_templates_manager/unit_template_minis.jsx

```diff
@@ -102,8 +102,24 @@ export default class UnitTemplateMinis extends React.Component {
     )
   };
 
+  getModelCardsByType(models, type) {
+    const filteredModels = models.filter(model => {
+      const { unit_template_category, id } = model
+      if(id === CREATE_YOUR_OWN_ID) {
```

Comment:
I would do this as `if id === CREATE_YOUR_OWN_ID && type === WHOLE_CLASS_LESSONS` rather than nesting the second `if`